manifest.json

```diff
@@ -1,15 +1,22 @@
 {
+  "$schema": "https://json.schemastore.org/chrome-manifest",
```

Comment:
Adding the $schema is out of scope for MV3 fork

Reply:
It's there for my sake \:), I'll remove at the end.